### PR TITLE
DM-45618: Update ApPipe clustering configs.

### PR DESCRIPTION
**Pull request by erinleighh**

### CODE REVIEW

File: bps/clustering/clustering_ApPipeWithFakes.yaml [hunk 1]
```diff
@@ -19,14 +19,19 @@
...
-    pipetasks: coaddFakes
-    dimensions: tract,patch
   diffim:
-    pipetasks: processVisitFakes,retrieveTemplate,subtractImages,detectAndMeasure,filterDiaSrcCat,rbClassify,transformDiaSrcCat,diaPipe,fakesMatch
+    pipetasks: initialPviCore,retrieveTemplate,inject_visit,subtractImages,detectAndMeasure,diffimTaskCore,filterDiaSrcCat,analyzeTrailedDiaSrcCore,sampleSpatialMetrics,diffimTaskPlots,rbClassify,transformDiaSrcCat
     dimensions: visit,detector
-    equalDimensions: visit:exposure
+  association:
+    pipetasks: getRegionTimeFromVisit,mpSkyEphemerisQuery,loadDiaCatalogs,diaPipe
+    dimensions: detector,group
```
Comment (isullivan) [inline bps/clustering/clustering_ApPipeWithFakes.yaml:30]: This should be `visit, detector`

**Review by isullivan — CHANGES_REQUESTED** [review]
Please update `clustering_ApPipe.yaml` with the same clusters, where any of the tasks exist.

File: bps/clustering/clustering_ApPipeWithFakes.yaml [hunk 2]
```diff
@@ -19,14 +19,19 @@
...
+    equalDimensions: visit:group
+  diaSrcDetectorAnalysis:
+    pipetasks: analyzeAssocDiaSrcCore,injectedMatch,analyzeDiaFakesDetectorVisitCore
+    dimensions: visit,detector
+  diaSrcVisitAnalysis:
+    pipetasks: consolidateMatchDiaSrc,analyzeDiaFakesVisitCore
+    dimensions: visit
```
Comment (kherner) [inline bps/clustering/clustering_ApPipeWithFakes.yaml:37]: I think it's complaining about a missing newline at the end here.

**Review by kherner — CHANGES_REQUESTED** [review]
Ian already caught the needed detector,group -> visit,detector change on L30.

**Review by kherner — APPROVED** [review]

File: bps/clustering/clustering_ApPipe.yaml [hunk 3]
```diff
@@ -23,6 +23,12 @@ cluster:
     dimensions: visit,detector
     equalDimensions: visit:exposure
   diffim:
-    pipetasks: retrieveTemplate,subtractImages,detectAndMeasure,filterDiaSrcCat,rbClassify,transformDiaSrcCat,diaPipe
+    pipetasks: initialPviCore,retrieveTemplate,subtractImages,detectAndMeasure,diffimTaskCore,filterDiaSrcCat,sampleSpatialMetrics,diffimTaskPlots,rbClassify,transformDiaSrcCat
```
Comment (isullivan) [inline bps/clustering/clustering_ApPipe.yaml:26]: `initialPviCore` should move to `singleFrame`

File: bps/clustering/clustering_ApPipeWithFakes.yaml [hunk 4]
```diff
@@ -19,14 +19,19 @@
...
-    pipetasks: isr,characterizeImage,calibrate
+    pipetasks: isr,calibrateImage
     dimensions: visit,detector
     equalDimensions: visit:exposure
-  coaddFakes:
-    pipetasks: coaddFakes
-    dimensions: tract,patch
   diffim:
-    pipetasks: processVisitFakes,retrieveTemplate,subtractImages,detectAndMeasure,filterDiaSrcCat,rbClassify,transformDiaSrcCat,diaPipe,fakesMatch
+    pipetasks: initialPviCore,retrieveTemplate,inject_visit,subtractImages,detectAndMeasure,diffimTaskCore,filterDiaSrcCat,sampleSpatialMetrics,diffimTaskPlots,rbClassify,transformDiaSrcCat
```
Comment (isullivan) [inline bps/clustering/clustering_ApPipeWithFakes.yaml:26]: As above, `initialPviCore` should move to `singleFrame`

**Review by isullivan — APPROVED** [review]
One minor change to both files.

**Pull request merged**